styles/index.css

```diff
@@ -61,6 +75,7 @@ form {
     margin: 0 auto;
     max-width: 600px;
     padding: 30px;
+    box-shadow: 10px 15px gray;
```

Comment:
this is also good. although your box shadow seems too thick 🙂

Reply:
i have adjusted it to 5px 5px